tests-e2e/lists_test.go

```diff
@@ -994,7 +1001,7 @@ func TestBumpEventTypesHandling(t *testing.T) {
 				t.Fatalf("expected no ops while waiting for receipt: %s", err)
 			}
 			matchReceipt := m.MatchReceipts(room2, []m.Receipt{{
-				EventID: charlieJoinEventID,
+				EventID: aliceLastSeenEventID,
```

Comment:
I was kinda surprised this worked before. But I guess the receipt event IDs are opaque to the homeserver?

Reply:
I guess so! :S